src/Sarif.UnitTests/JsonTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Microsoft.CodeAnalysis.Sarif.Readers;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class JsonTests
+    {
+        public JsonTests()
+        {
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings
+            {
+                ContractResolver = SarifContractResolver.Instance
+            };
+        }
+    }
+}
```

Comment:
Funny you should be doing this; I did the same thing in a branch recently. I suggest you move the `GetJson` method that's duplicated in a few test classes into this base class as well.